packages/gatsby/src/commands/plugin/configure.ts

```diff
@@ -22,16 +31,36 @@ export async function run(): Promise<void> {
     .map(name => {
       const option = options[name]
       const supported = supportedOptionTypes.includes(option.type)
+      // TODO: Don't hard code this.
+      // Maybe we just allowlist a bunch of options like accessToken, password, token?
+      const isSensitive = name === `accessToken`
```

Comment:
I think the way would be for it to be options that the schema marks as being stored in a .env file, which we were planning to do with an extension.

Reply:
I don't really want to block any of this on the schema extension 😬  We should definitely do it in the future, but for now IIRC the only plugin with secrets and a plugin schema is the Contentful plugin, so let's just leave this hard-coded and tackle that soon.